controllers/update_status.go

```diff
@@ -599,8 +598,7 @@ func validateInstance(r *FoundationDBClusterReconciler, context ctx.Context, clu
 	var needsSidecarConfInConfigMap bool
 	for _, container := range instance.Pod.Spec.Containers {
 		if container.Name == "foundationdb" {
-			image := strings.Split(container.Image, ":")
-			version, err := fdbtypes.ParseFdbVersion(image[len(image)-1])
+			version, err := fdbtypes.ParseFdbVersion(cluster.Status.RunningVersion)
```

Comment:
I think this is fine in the short-term but wen probably want to use the version of the process group (not yet implemented).